zulip_bots/zulip_bots/bots/converter/converter.py

```diff
@@ -46,11 +49,11 @@ def usage(self):
                all supported units.
                '''
 
-    def handle_message(self, message, bot_handler):
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
```

Comment:
You should be able to tell from the way in which `message` is used, that it can be typed more strongly than this (particularly in how the same passed-through variable is used below), and also from `lib.py`.